Constitution.md

```diff
@@ -97,32 +101,102 @@ The Terms of Office for all Executive Positions at CSSA will last for at most on
+15. Perform any other duties incidental to the office of the President or directed by Council.
+
+### Director of Finances and Internal Affairs
+
+1. Act as a signing authority for CSSA.
+2. Providing a brief report on CSSA finances and expenses at each meeting.
+3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
+4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
+5. Ensure the Council is kept updated on ongoing council initiatives.
+6. Perform a year-end audit at the end of the CSSA Fiscal Year.
+7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
+8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
+9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
+10. Ensure that the CSSA lounge is a safe and inclusive space.
+11. Lead the Financial Accounability Committee with up to 3 committee members.
+12. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+
+### Director of Lounge Affairs
+
+1. Act as a signing authority for CSSA.
+2. Create a cleaning schedule for the CSSA Lounge.
+3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
+4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
+5. Ensure that the CSSA lounge is a safe and inclusive space.
+6. Lead the CSSA lounge supervisors and work with council in appointing them.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Student Affairs
+
+1. Act as a liaison for the students towards the Department of Computer Science, Faculty of Science and UMSU.
+2. Act as a liaison between Students and Council.
+3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
+4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
+5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
+6. Ensure that the CSSA lounge is a safe and inclusive space.
+7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.
+
+### Director of Events
+
+1. Be responsible for planning the events for the Computer Science community.
+2. Coordinate team-building events for Council.
+3. Coordinate a minimum of one event or academic initiative in each term, as per the discretion of the Council.
+4. Ensure there are an adequate number of social and academic events.
+5. Create posters and advertisements for all CSSA events.
```

Comment:
Again this is super specific. Is this up to the events exec or is this for the events committee?

Reply:
It is something they are accountable for, but up to them to decide how they want to split or delegate their responsibility to others.